src/main/java/org/maxgamer/quickshop/shop/ContainerShop.java

```diff
@@ -21,6 +21,7 @@
 
 import com.lishid.openinv.OpenInv;
 import io.papermc.lib.PaperLib;
+import jdk.internal.net.http.common.Utils;
```

Comment:
not a good idea to use internal utils

Reply:
It's an unused imports lol

Reply:
Good catch, I must've alt-entered accidentally and it imported that without me noticing.